src/components/forms/validator.ts

```diff
@@ -91,6 +93,20 @@ export const mustBeEthereumAddress = memoize((address: string): ValidatorReturnT
   return result
 })
 
+export const checkNetworkPrefix = (value: string): ValidatorReturnType => {
+  const errorMessage = "The current network doesn't match the given address"
```

Comment:
Should we have a separate error message for prefixes that aren't valid shortNames?

Reply:
Done!

I added the `Unsupported network prefix` error label in the validator:
```javascript
  const errorMessage = isValidShortChainName(prefix)
    ? "The current network doesn't match the given address"
    : 'Unsupported network prefix'
```
And also added a unit test to this new case:
```javascript

    it('Validates unsupported shortNames', () => {
      const unsupportedPrefixedAddress = 'xxxx:0x2D42232C03C12f1dC1448f89dcE33d2d5A47Aa33'

      renderAddressInputWithinForm()

      // invalid address network prefix
      fireEvent.change(screen.getByTestId(fieldTestId), { target: { value: unsupportedPrefixedAddress } })

      // input value with the network prefix
      expect((screen.getByTestId(fieldTestId) as HTMLInputElement).value).toBe(unsupportedPrefixedAddress)

      // show prefix error
      expect(screen.queryByText('Unsupported network prefix')).toBeInTheDocument()
    })


```